lib/app/modules/alarmChallenge/bindings/alarm_challenge_binding.dart

```diff
@@ -8,5 +9,8 @@ class AlarmChallengeBinding extends Bindings {
     Get.lazyPut<AlarmChallengeController>(
       () => AlarmChallengeController(),
     );
+    Get.put<HapticFeebackService>(
```

Comment:
Is there any reason you preferred put over lazyPut? I'd like to get your insights on this as well.